cbor.js

```diff
@@ -344,13 +476,15 @@ function decode(data, tagger, simpleValue) {
         }
         return readArrayBuffer(length);
       case 3:
-        var utf16data = [];
+        var newStr = "";
         if (length < 0) {
-          while ((length = readIndefiniteStringLength(majorType)) >= 0)
-            appendUtf16Data(utf16data, length);
-        } else
-          appendUtf16Data(utf16data, length);
-        return String.fromCharCode.apply(null, utf16data);
+          while ((length = readIndefiniteStringLength(majorType)) >= 0) {
+            newStr = appendUtf16Data(newStr, length);
+          }
+        } else {
+          newStr = appendUtf16Data(newStr, length);
+        }
+        return newStr;
```

Comment:
do you have some performance numbers for this change? or is there an other reason for this change?